platform/posix/transport/src/mbedtls_pkcs11_posix.c

```diff
@@ -85,8 +81,12 @@ static const char * pNoLowLevelMbedTlsCodeStr = "<No-Low-Level-Code>";
 
 /**
  * @brief Debug logging level to use for MbedTLS.
+ *
+ * @note The default value of 0 disables MbedTLS logging.
+ * See <https://tls.mbed.org/api/debug_8h.html#a6629362e96b43725ace95c8ff01d9985>
+ * for valid values.
  */
-#define MBEDTLS_DEBUG_LEVEL    0
+#define MBEDTLS_DEBUG_LOG_LEVEL    0
```

Comment:
It would be better to expose this at least in the header file of the transport library, if not `demo_config.h` file, so that customers can find a way to enable mbedTLS logging more easily. However, it would still be valuable to mention this in the README under a Troubleshooting section.

Reply:
Not sure about the README, as this is only for this transport interface, but that is a different concern. I'll move it to the header file.